drape_frontend/drape_engine.cpp

```diff
@@ -477,4 +476,11 @@ void DrapeEngine::SetKineticScrollEnabled(bool enabled)
                                   MessagePriority::High);
 }
 
+void DrapeEngine::SetTimeInBackground(double time)
+{
+  m_threadCommutator->PostMessage(ThreadsCommutator::RenderThread,
+                                  make_unique_dp<SetTimeInBackgroundMessage>(time),
```

Comment:
Так может лучше не время отправлять, а сразу нужное событие, которое должно произойти по условию этого времени?

> On 14.4.2016, at 11.41, Roman Kuznetsov <notifications@github.com> wrote:
> 
> In drape_frontend/drape_engine.cpp:
> 
> > @@ -477,4 +476,11 @@ void DrapeEngine::SetKineticScrollEnabled(bool enabled)
> >                                    MessagePriority::High);
> >  }
> >  
> > +void DrapeEngine::SetTimeInBackground(double time)
> > +{
> > +  m_threadCommutator->PostMessage(ThreadsCommutator::RenderThread,
> > +                                  make_unique_dp<SetTimeInBackgroundMessage>(time),
> Нельзя, отрисовка ничего не знает про бэкграунд, только по косвенным признакам может это предполагать.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>